components/match2/wikis/counterstrike/match_legacy.lua

```diff
@@ -0,0 +1,245 @@
+---
```

Comment:
please change double quotes to single quotes everywhere (as long as not inside a string)